packages/onchainkit/src/swap/types.ts

```diff
@@ -309,52 +324,57 @@ export type SwapReact = {
   onStatus?: (lifecycleStatus: LifecycleStatus) => void;
   /** An optional callback function that exposes the transaction receipt */
   onSuccess?: (transactionReceipt: TransactionReceipt) => void;
-  /** Title for the Swap component. (default: "Swap") */
-  title?: ReactNode;
-  /** Header left content for the Swap component (eg. back button) */
-  headerLeftContent?: ReactNode;
 } & (
   | {
-      /** When React children are provided
-       * swappableTokens, toToken, and fromToken should be passed to the SwapAmountInput component
-       * disabled should be passed to the SwapButton component
-       **/
-      children: ReactNode;
+      /** When children is provided, swappableTokens, toToken, and fromToken are not required */
+      children: ({
+        componentThemeClassName,
+      }: {
+        componentThemeClassName: string;
+      }) => ReactNode;
```

Comment:
issue: This works the same as the render prop, but with a different name. Can we use `render` for consistency?